### ISSUE TITLE
Alternative ways to communicate

### ISSUE THREAD

**Issue by fadamakis** [devstaff-crete/DevStaff-Heraklion#21]
Github issues are terrible for what we are trying to do.

Some alternatives are:
- Meetup's [discussion](http://www.meetup.com/DevStaff-A-Developer-Community-Gathering-In-Crete/messages/boards/) and [poll](http://www.meetup.com/DevStaff-A-Developer-Community-Gathering-In-Crete/polls/) features :heart: 
- A google group
- A forum since we have our own domain
- A slack chat
- A subreddit
- ...

**Comment by padeler** [comment]
I think you are right. Maybe we should keep this until the next meetup and switch then?

**Comment by zakkak** [comment]
Regarding topics I also believe we need a platform that supports polls (A subreddit would be nice).

I am also in favor of using mailing-lists for general discussions (that do not require polling), however a subreddit seems sufficient for both cases.

**Comment by DaKnOb** [comment]
Basically only voting is the problem because votes have to be counted manually. Maybe a subreddit is the best.

**Comment by gsaslis** [comment]
i personally agree with @daknob, but i'd very much like to hear more ppl's
views:

* I've used google groups for the AgileCrete and JCrete unconferences, and
in a couple of other cases.. sooo many times has there been a case where
someone's been saying 'I missed the notification for that' / "didn't get
that e-mail" / etc. so I would prefer !googlegroups.

* For polls, you're right guys, GitHub isn't as suitable for voting (for
example, GitLab has a thumbs-up-counter feature that makes it easy to keep
track of votes) so we'd probably need to find a better solution..
* Would the meetup polls also allow for discussion on each topic, just like
we're having in the issues here?

* Slack is great. Slack is also expensive though, unless you don't mind it
only remembering the last 10K messages shared in your team (and we reached
that pretty fast w/ a team of 10 ppl at work).

* For forums - tbh, i've never been a huge fan.
* Subreddit - no idea what that is / how it works...

Perhaps instead of judging each solution, we should first come up with a
list of requirements for our communication platform:
* it needs to be free and open, so that everyone can join
* I like @mentions, due to notifications (this is also a feature I like in
slack) -- I'd propose the new tool we find supports them, if you agree
* voting features would be nice (e.g. Trello boards also support voting I
think)
* email integration is a must, I guess, a native app for push notifications
would be nice (?)

please feel free to add your own!


-- 
Yorgos Saslis
Software Engineer

**Comment by sbehnsen** [comment]
I agree that there are probably better/other solutions out there. Probybly Google would be the easiest solution as most likely many of us already use it somewhere.
However I personally find the very technical solution here interesting because it kind a make administration of ourselves a solftware project... :-)

**Comment by zakkak** [comment]
Today I also noticed that we need a way to discuss about past topics and new findings/advances in the corresponding area.

**Comment by gsaslis** [comment]
Good point.. I guess one way is adding a comment in the [meetup page](
http://www.meetup.com/DevStaff-A-Developer-Community-Gathering-In-Crete/events/223500458/
).

Another would be to submit a PR [here](
https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/meetups/meetup01-Git/README.md)
.. ?

-- 
Yorgos Saslis
Software Engineer

**Comment by kabitakis** [comment]
I believe voting can work as is since there are no blocking/critical issues addressed during its first week of use. However, discussing topics/issues is really important for this community to develop and we should find a (better) way to communicate efficiently. I was also thinking of ways of (micro)blogging for the community. I'll commit some docs/specs describing what we need to do and I hope for some help. Based on those docs/specs we could arrange a meeting between people interested in getting involved in the process. What do you think?

**Comment by sbehnsen** [comment]
I am starting to have an issue with the voting 'as is'. the amount of emails generated is extreme. Of course I can put a filter in my email to automatically delete the emails from githup-devstaff but that would probably also delete usefull Information...
For this reason a BBS is, in my opinion, a much better solution... (and better voting Systems are provided as well...)

**Comment by eirinikp** [comment]
You can put a filter that automatically labels it and archives the email, instead of deleting it

**Comment by sbehnsen** [comment]
I know, there is a lot I can filter, but there are some issues:
1. It is not always easy to be sure that you filter only what you want, so you may filter something in error
2. If you don't delete the messages you end up with thousands of messages in storrage...
3. It is not pracitcal to have thousands of messages in storrage if you have to look for something later, since all messages from Github look very similar.
4. If I have to look though the stored messages, than I may as well have them in my inbox...
So either the messages are prevented at the source, or I will have to install some sort of filter...
(and before you say it, yes of course before deleting, I would filter into a storrage to see if the filtering seems to do what I want, and then delete at a later time, going into direct delete only after a test period...)

BUT:
If everybody else is happy to receive all these most the time useless messages, I will shut up, but I think this is quite unlikely. I am interested in valid comments on a post of myself, but I am not interested for example to see every vote happening. I believe that is probably the case for most people, that work a lot with email. I for my part have very many emails every day, and I am working hard on preventing unwanted emails to reach my inbox...

**Comment by fadamakis** [comment]
May I suggest once more the poll feature in meetup.com? 
It's way more clean and elegant as you can see [here](http://www.meetup.com/Agile-Greece/polls/926422/)

**Comment by DaKnOb** [comment]
A solution for GitHub is *not* to watch the repository and then from that point you only receive e-mails from issues where you're mentioned or you commented something. In each one of these you have the option to Unsubscribe so you can actually get only e-mails from issues you want. If you don't want any e-mails, there's also this option too.

**Comment by kabitakis** [comment]
Ok then! I suppose there are some github fans as well, so, can we record our suggestions here and then run a poll to decide on the way of voting. @fadamakis since you opened this issue, can you put all the suggestions together on a table in your first comment so we can have an overview?

**Comment by eirinikp** [comment]
I guess that sbehnsen means that he only wants to delete the +1 emails, but he wants to receive the rest of them even when he doesn't comment on them. 
For me the filter and archive solution works, but apparently not for everyone. 
However using meetup for the polls and github for the rest of the discussion of the same topic seems too much, I believe we should choose one. Also, I am not sure that even when the +1s will stop, the emails will be much less

**Comment by kabitakis** [comment]
Ok, I don't know what's going to happen with our voting options but I developed a small node.js project to count the topic votes. You can check it here: https://powerful-sands-5380.herokuapp.com/
Nothing fancy, it's WIP and I intend to add many features and more charts, however it's an automated way to count the votes (for now). You can also get a json object using: https://powerful-sands-5380.herokuapp.com/api/issues

**Comment by zakkak** [comment]
Nice :+1:

**Comment by eirinikp** [comment]
:+1: :)

**Comment by webdevgr** [comment]
:+1:

**Comment by kabitakis** [comment]
I was thinking we should close this issue and open some more specific ones in case we still have communication problems. So, don't be afraid to open a new issue if anything concerns you.
